client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -258,14 +258,15 @@ export const isCustomsFormRequired = createSelector(
 				'An origin address phone number is required for this shipment.'
 			);
 		} else if (
+			'US' === country &&
```

Comment:
Why do we have this check here now?

Reply:
I thought 10 digit phone number only related to US phone number?